migration/validator-activation.md

```diff
@@ -0,0 +1,76 @@
+
+### **Step 1: Add Validator Data into the Configuration File**
+
+Once your PoS client is set up, follow these steps:
+
+1. Check if the `$HOME/.nimiq` folder exists:
+    1. If it exists, open the configuration file in `$HOME/.nimiq/client.toml` and locate the dedicated section for validators at the end
+    2. If it doesn’t exist, create the `$HOME/.nimiq` folder and inside, create a `client.toml` file based on this [example](https://github.com/nimiq/core-rs-albatross/blob/albatross/lib/src/config/config_file/client.example.toml)
+2. Paste your **validator data** into the validator section (public key, signing key, voting key)
+3. Set the **network** is set to `main-albatross` and the `sync_mode` to `full` or `history`. Only full or history nodes are eligible to become validators
+4. Review and adjust all the other sections of `client.toml` and ensure that the parameters are configured according to your specific setup (paths, keys, and any optional features you want to enable). Adjust these settings as needed to fit your node's configuration
+
+### **Step 2: Download the PoW Chain Database Snapshot (Optional)**
+
+If your PoW client is not synced or you’re setting it up from scratch, you can download a database snapshot to accelerate the synchronization process. This snapshot is particularly useful before running the Activation Tool, as it eliminates the need to sync the entire PoW chain from scratch. After downloading the snapshot, only a small portion of the chain needs to be synced, allowing your node to reach consensus more quickly:
+
+- **IPFS**: Download the full PoW chain snapshot [here](https://ipfs.nimiq.io/ipfs/QmRKvFVpTdXagvgZG5cF9qdz13x9DkZhUvwXAS5YMaqTfu?filename=pow-main-full-consensus.zip)
+- **Torrent**: Alternatively, download the snapshot via Torrent [here](https://repo.nimiq.com/torrents/nimiq-pow-main-full-consensus.torrent)
+
+After downloading the snapshot, the database file must be placed in the directory from where you are running your PoW node. Once in place, follow steps [3](#step-3-prepare-the-pow-client-for-the-transition) and [4](#step-4-run-the-activation-tool). After your node reaches consensus, continue with the Activation Tool to complete the process.
+
+### **Step 3: Prepare the PoW Client for the Transition**
+Ensure the **PoW client is running with RPC enabled** on a distinct port from the PoS client. Configure the PoW client’s RPC settings in its configuration file by adding your validator address and key pair. Refer to [**this example**](https://github.com/nimiq/core-js/blob/master/clients/nodejs/sample.conf#L163).
+
+Run the following command to start the PoW client:
+
+```bash
+    node clients/nodejs/index.js --dumb --network=main --rpc=8648 --wallet-seed=private_keypublic_key
+```
+
+The `private_keypublic_key` is the concatenation of your private key and public key in that order
```

Comment:
is the concatenation of your validator address private key and public key in that order, this is necessary because the activation tool is going to send and sign transactions in behalf of your validator to indicate readiness.